semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/SystemIncubatingAttributes.java

```diff
@@ -25,10 +25,6 @@ public final class SystemIncubatingAttributes {
   /** The device identifier */
   public static final AttributeKey<String> SYSTEM_DEVICE = stringKey("system.device");
 
-  /** The disk operation direction */
-  public static final AttributeKey<String> SYSTEM_DISK_DIRECTION =
```

Comment:
This might be another set of examples of deleted semconv?

Reply:
Yeah I think the process to switch from deleting to deprecating and retaining forever (or at least a while) is a relatively new thing. We just started to see it in 1.24.0, and I expect it will be the norm going forward.

@lmolkova / @trask may be able to confirm.